h2o-algos/src/test/java/hex/coxph/CoxPHTest.java

```diff
@@ -264,12 +276,15 @@ public void testCoxPHSingleNodeMode() {
       parms._response_column = "event";
       parms._ignored_columns = new String[]{"id", "year", "surgery", "transplant"};
       parms._ties = CoxPHModel.CoxPHParameters.CoxPHTies.efron;
+      parms._skip_scoring = true;
```

Comment:
this just makes the test pass? why is it necessary?

Reply:
+1, should be explained here

Reply:
Explained.

Reply:
The way I undestand this is that when user uses the params & input above, the algo will crash, is that correct? if yes, I dont thinkg that is right and needs to be fixed

Reply:
No, the algo will not crash in such a situation.

What caused crash in the test is `sys.ai.h2o.debug.checkRunLocal` system property used in this test, that probably shouln't be used in prod.
I made `_skip_scoring` package private to make it less likely to be used by accident.